### PR TITLE
fix: Use data attributes in pl-xss-safe

### PR DESCRIPTION
**Pull request by dwang**
Instead of templating the student-provided content to a variable in the `pl-xss-safe` element, it is passed as a data attribute (`data-code`). This removes an XSS vector where a closing `</script>` tag gets interpreted, so the content is never sanitized.

For consistency, the language is also passed as a data attribute (`data-language`).

@jonatanschroeder

### CODE REVIEW

**Comment by dwang** [comment]
I have reviewed and hereby sign the CLA

**Review by nwalters512 — APPROVED** [review]
Tested locally - this is looking good! @jonatanschroeder I'll wait for your review before merging - security things like this could use as many eyes as possible.

**Review by jonatanschroeder — APPROVED** [review]

**Pull request merged**
